x/circuit/README.md

```diff
@@ -87,7 +87,6 @@ https://github.com/cosmos/cosmos-sdk/blob/main/proto/cosmos/circuit/v1/tx.proto#
 This message is expected to fail if:
 
 * the granter is not an account with permission level `LEVEL_SUPER_ADMIN` or the module authority
```

Comment:
It would be nice to precise the granter can be governance as well and that gov is considered as `LEVEL_SUPER_ADMIN` as well.